src/Microsoft.PowerShell.Commands.Management/commands/management/Process.cs

```diff
@@ -2992,6 +2992,7 @@ public ProcessCommandException(string message, Exception innerException)
         /// <param name="info"></param>
         /// <param name="context"></param>
         /// <returns>Constructed object.</returns>
+        [ObsoleteAttribute("Formatter-based serialization is obsolete and should not be used.")]
```

Comment:
We already used this message for obsolete attribute:
> [Obsolete("Legacy serialization support is deprecated since .NET 8", DiagnosticId = "SYSLIB0051")] 

(see #19696)
So, please re-use the same message.